com.unity.multiplayer.mlapi/Editor/ProfilerModuleOverride/NetworkProfilerOverride.cs

```diff
@@ -0,0 +1,249 @@
+
+        [System.Serializable]
+        public class MLAPIModules
+        {
+            public List<MLAPIProfilerModuleData> m_Modules;
+        }
+
+        private static List<MLAPIProfilerCounter> CreateRPCCounters()
+        {
+            var list = new List<MLAPIProfilerCounter>();
+            list.Add(new MLAPIProfilerCounter() { m_Name = ProfilerConstants.NumberOfRPCsSent.ToString(), m_Category = ProfilerCategory.Network.Name });
+            list.Add(new MLAPIProfilerCounter() { m_Name = ProfilerConstants.NumberOfRPCsReceived.ToString(), m_Category = ProfilerCategory.Network.Name });
+            list.Add(new MLAPIProfilerCounter() { m_Name = ProfilerConstants.NumberOfRPCBatchesSent.ToString(), m_Category = ProfilerCategory.Network.Name });
+            list.Add(new MLAPIProfilerCounter() { m_Name = ProfilerConstants.NumberOfRPCBatchesReceived.ToString(), m_Category = ProfilerCategory.Network.Name });
+            list.Add(new MLAPIProfilerCounter() { m_Name = ProfilerConstants.NumberOfRPCQueueProcessed.ToString(), m_Category = ProfilerCategory.Network.Name });
+            list.Add(new MLAPIProfilerCounter() { m_Name = ProfilerConstants.NumberOfRPCsInQueueSize.ToString(), m_Category = ProfilerCategory.Network.Name });
+            list.Add(new MLAPIProfilerCounter() { m_Name = ProfilerConstants.NumberOfRPCsOutQueueSize.ToString(), m_Category = ProfilerCategory.Network.Name });
+            return list;
+        }
+
+        private static List<MLAPIProfilerCounter> CreateOperationsCounters()
+        {
+            var list = new List<MLAPIProfilerCounter>();
+            list.Add(new MLAPIProfilerCounter() { m_Name = ProfilerConstants.NumberOfConnections.ToString(), m_Category = ProfilerCategory.Network.Name });
+            list.Add(new MLAPIProfilerCounter() { m_Name = ProfilerConstants.ReceiveTickRate.ToString(), m_Category = ProfilerCategory.Network.Name });
+            return list;
+        }
+
+        private static List<MLAPIProfilerCounter> CreateMessagesCounters()
+        {
+            var list = new List<MLAPIProfilerCounter>();
+            list.Add(new MLAPIProfilerCounter() { m_Name = ProfilerConstants.NumberOfNamedMessages.ToString(), m_Category = ProfilerCategory.Network.Name });
+            list.Add(new MLAPIProfilerCounter() { m_Name = ProfilerConstants.NumberOfUnnamedMessages.ToString(), m_Category = ProfilerCategory.Network.Name });
+            list.Add(new MLAPIProfilerCounter() { m_Name = ProfilerConstants.NumberBytesSent.ToString(), m_Category = ProfilerCategory.Network.Name });
+            list.Add(new MLAPIProfilerCounter() { m_Name = ProfilerConstants.NumberBytesReceived.ToString(), m_Category = ProfilerCategory.Network.Name });
+            list.Add(new MLAPIProfilerCounter() { m_Name = ProfilerConstants.NumberNetworkVarsReceived.ToString(), m_Category = ProfilerCategory.Network.Name });
+            return list;
+        }
+
+        private delegate List<MLAPIProfilerCounter> CounterFunction();
```

Comment:
nit, name here is a little fuzzy. it could be more specific, like `CounterListFactoryDelegate`

Reply:
Has been renamed